deepchem/feat/graph_data.py

```diff
@@ -2,73 +2,69 @@
 import numpy as np
 
 
-class MoleculeGraphData:
-  """MoleculeGraphData class
+class GraphData:
+  """GraphData class
 
   This data class is almost same as `torch_geometric.data.Data 
   <https://pytorch-geometric.readthedocs.io/en/latest/modules/data.html#torch_geometric.data.Data>`_.
 
   Attributes
   ----------
-  node_features : np.ndarray
+  node_features: np.ndarray
     Node feature matrix with shape [num_nodes, num_node_features]
-  edge_index : np.ndarray, dtype int
+  edge_index: np.ndarray, dtype int
     Graph connectivity in COO format with shape [2, num_edges]
-  targets : np.ndarray
-    Graph or node targets with arbitrary shape
-  edge_features : np.ndarray, optional (default None)
+  edge_features: np.ndarray, optional (default None)
```

Comment:
One generalization we might want to consider is adding `pair_features`. The `WeaveFeaturizer` class has pair features for all pairs of atoms, even those which aren't connected by an edge

Reply:
I seem that we can define the pair features using the edge_idx and edge_features, but I'm not deeply understanding WeaveNet Implementation. 

I think it is better to add the attribute when we start to building the WeaveModel using new graph data.

Reply:
Sounds good! Probably better to generalize with a concrete use case in hand